onelogin_aws_cli/credentials.py

```diff
@@ -83,7 +83,7 @@ class UserCredentials(object):
     SERVICE_NAME = "onelogin-aws-cli"
 
     def __init__(self, config: Section):
-        self.username = config['username']
+        self.username = config.get('username')
```

Comment:
What happens if there's no username. Do we prompt the user, or fail a bit nicer?

Reply:
We prompt them =)